influxql/parser.go

```diff
@@ -442,7 +444,7 @@ func (p *Parser) parsePrivilege() (Privilege, error) {
 
 // parseSelectStatement parses a select string and returns a Statement AST object.
 // This function assumes the SELECT token has already been consumed.
-func (p *Parser) parseSelectStatement() (*SelectStatement, error) {
+func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, error) {
```

Comment:
`found FROM, expected INTO at line 1, char 63` is pretty clear to me  :)  The other way reports what is missing but not where it goes in the statement.